src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -352,6 +352,7 @@ private void AppendOutput(StringBuilder propertyValues, string append)
         public bool ApplyPackageParametersToDependencies { get; set; }
         public bool ApplyInstallArgumentsToDependencies { get; set; }
         public bool IgnoreDependencies { get; set; }
+        public bool IgnoreHttpCache { get; set; }

```

Comment:
This is again a double negative here. Same as #3727, we should probably turn this the other way around.
Ie, `IncludeHttpCache`, or `UseHttpCache`, or maybe `EnableHttpCache`.

Reply:
I have done as you suggested here.